impl/client/src/main/javascript/web/dojo/pentaho/common/Dialog.js

```diff
@@ -174,6 +176,17 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
               this.popup.set('title',this.title);
             },
 
+            setupResponsiveness: function () {
+              //add "responsive" class to the dialog, along with any additional responsiveClasses defined
```

Comment:
It's more common to do `// Add` — space between `//` and first character and to capitalize a first letter.